intellij/build.gradle.kts

```diff
@@ -14,30 +12,52 @@ val ideProfile = IdeVersions.ideProfile(project)
 val publishToken: String by project
 val publishChannel: String by project
 
+val resharperDlls = configurations.create("resharperDlls") {
+    isCanBeConsumed = false
+}
+
 intellij {
     version = ideProfile.community.sdkVersion
     pluginName = "aws-toolkit-jetbrains"
     updateSinceUntilBuild = false
+
+    instrumentCode =  false
 }
 
-tasks.getByName<PrepareSandboxTask>("prepareSandbox") {
-    project.findProject(":jetbrains-rider")?.let {
-        from(tasks.getByPath(":jetbrains-rider:prepareSandbox"))
+tasks.prepareSandbox {
+    from(resharperDlls) {
+        into("aws-toolkit-jetbrains/dotnet")
     }
 }
 
-tasks.getByName<PublishTask>("publishPlugin") {
+tasks.publishPlugin {
     token(publishToken)
     channels(publishChannel.split(",").map { it.trim() })
 }
 
-tasks.named("check") {
+tasks.check {
     dependsOn(tasks.named("verifyPlugin"))
 }
 
+tasks.test {
+    enabled = false
+}
+
 dependencies {
+    implementation(project(":jetbrains-core"))
     implementation(project(":jetbrains-ultimate"))
     project.findProject(":jetbrains-rider")?.let {
         implementation(it)
+        resharperDlls(project(":jetbrains-rider", configuration = "resharperDlls"))
+    }
+}
+
+configurations {
+    // Make sure we exclude stuff we either A) ships with IDE, B) we don't use to cut down on size
```

Comment:
`we don't use to cut down on size`, can we comment which is which? aren't all of these included in the ide except netty we remove for size?

Reply:
ya, you are correct, will break it down by category